pkgs/development/python-modules/astropy/default.nix

```diff
@@ -123,11 +125,16 @@ buildPythonPackage rec {
   });
 
   nativeCheckInputs = [
+    hypothesis
     pytestCheckHook
     pytest-xdist
     pytest-astropy-header
-    pytest-astropy
+    pytest-doctestplus
+    pytest-remotedata
     threadpoolctl
+    # FIXME remove in 7.2.0
+    # see https://github.com/astropy/astropy/pull/18882
+    uncompresspy
```

Comment:
And this too - if this dependency is available a few more tests are being executed?

Reply:
The test was previously skipped because pytest-astropy disables "slow" tests (they aren't actually that slow).